api/docker/snapshot.go

```diff
@@ -134,6 +136,13 @@ func snapshotContainers(snapshot *portainer.Snapshot, cli *client.Client) error
 			runningContainers++
 		}
 
+		statusLen := len(container.Status)
+		if statusLen >= 9 && container.Status[statusLen - 9:] == "(healthy)" {
```

Comment:
I'd rather use `strings.Contains` here to search for a substring.

https://golang.org/pkg/strings/#Contains

Reply:
Sorry left for a vacation just as this came through :D

I've updated this in a fix commit, guess my original thoughts here were to avoid the additional import.  I defer to your judgement.